analyzer/command/apk/apk_test.go

```diff
@@ -162,14 +162,7 @@ func TestAnalyze(t *testing.T) {
 	}
 	analyzer := alpineCmdAnalyzer{}
 	for testName, v := range tests {
-		f, err := os.Open(v.apkIndexArchivePath)
-		if err != nil {
-			t.Fatalf("unexpected error: %s", err)
-		}
-		apkIndexArchive := &apkIndex{}
-		if err = json.NewDecoder(f).Decode(apkIndexArchive); err != nil {
-			t.Fatalf("unexpected error: %s", err)
-		}
+		apkIndexArchiveURL = v.apkIndexArchivePath
```

Comment:
Could you add a test using [httptest](https://golang.org/pkg/net/http/httptest/)? I'd like to test `https://` and `file://` both.